data/src/main/java/com/microsoft/azure/kusto/data/Utils.java

```diff
@@ -253,6 +253,9 @@ private static HttpPost setupHttpPostRequest(URI uri, String payload, InputStrea
 
         request.addHeader(HttpHeaders.ACCEPT_ENCODING, "gzip,deflate");
         request.addHeader(HttpHeaders.ACCEPT, "application/json");
+
+        String KUSTO_API_VERSION = "2019-02-13";
```

Comment:
I feel like this should be defined in the `pom`.

Reply:
Why? and how?

Reply:
Because this is configuration, not code.

https://stackoverflow.com/questions/11500533/access-maven-properties-defined-in-the-pom

Reply:
Added TODO as discussed